### DISCUSSION TITLE
Missing popularity filter for block themes on WP.org could be one reason for low block theme adoption

### DISCUSSION THREAD

**Discussion by Ren2049** [WordPress/gutenberg#54327] [General]
It's currently impossible to filter block themes by popularity.

![filter](https://github.com/WordPress/gutenberg/assets/133038357/21356a50-a32a-4504-b671-ca52f717385a)

But popularity/social proof is one of the most important factors for users picking themes. As an user I want to see the the themes that are doing well and are picked up by other users and benefit of the side effects of a bigger theme community.

Ideally, both block themes and popularity should be selectable from the main theme window, not one or the other.

![popularity](https://github.com/WordPress/gutenberg/assets/133038357/944c552a-362d-4c07-9dc1-a13ff59f3e2e)

For example I wouldn have never known about the existence of Neve FSE (Neve is a populalr classic theme) if I wouldn't have scrolled down the list for a while. Neve FSE has 2000 installs which puts it among the most populalr block themes, but it's basically invisible if you're not searching for it.

**Comment by ClarusDignus** [comment]
Maybe create an issue instead of a discussion. You can use the 'Create issue from discussion' link in the sidebar to automatically do so.

**Comment by Ren2049** [comment]
@ClarusDignus I wasn't sure where to post it because it's not directly related to the plugin or core functionality, or if others even consider it as a problem.

@ndiego what do you think about this?

**Reply by ClarusDignus** [reply]
I consider it to be an obvious problem and imagine many others do too.

How about [creating a ticket on Make WordPress Core](https://core.trac.wordpress.org/newticket) as [WordPress Develop on GitHub](https://github.com/WordPress/wordpress-develop) suggests?

When creating the ticket, you can select WordPress.org Site as the component.

Please provide the link to your new ticket in this discussion.

**Reply by ClarusDignus** [reply]
If you want to get a sense of whether or not others consider it to be a problem, you could try asking on [WordPress on Reddit](https://www.reddit.com/r/wordpress) or try asking [Jamie Marsland of PootlePress](https://twitter.com/pootlepress?lang=en) to post a poll on the matter on Twitter.

**Reply by Ren2049** [reply]
@ClarusDignus Gonna try it through wordpress-develop and update this comment when it's done.

You probably found this issue through my comment on Jamies site. I've already emailed him to unblock the comment, and then we had a short discussion about the other topic in the comment. I guess he's aware of it already.

**Reply by ClarusDignus** [reply]
Yeah, I saw your comment on Pootlepress.
